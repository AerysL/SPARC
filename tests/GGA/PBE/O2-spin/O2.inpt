SPIN_TYP: 2
CELL: 21 18 18
FD_GRID: 105 90 90
FD_ORDER: 12
BC: P P P
EXCHANGE_CORRELATION: GGA_PBE
ELEC_TEMP_TYPE: fermi-dirac
ELEC_TEMP: 315.775131
TOL_SCF: 1e-6
TOL_POISSON: 1e-8
TOL_PSEUDOCHARGE: 1e-8
NSTATES: 10
MIXING_VARIABLE: density
CALC_STRESS: 1
PRINT_FORCES: 1
PRINT_ATOMS: 1
