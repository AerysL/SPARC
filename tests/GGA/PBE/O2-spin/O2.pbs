#PBS -N O2
#PBS -q mps
#PBS -o output.sparc
#PBS -j oe
#PBS -l nodes=1:ppn=40
#PBS -l mem=10gb
#PBS -l walltime=1:00:00
MYPROCS=`wc -l < ${PBS_NODEFILE}`
module purge
module load intel/15.0
module load mvapich2/2.1
module load fftw/3.3.4
module load mkl/11.2

ulimit -s unlimited

cd ${PBS_O_WORKDIR}
echo $PWD
echo "======================================================================="
mpirun -env MV2_ENABLE_AFFINITY=1 -env MV2_CPU_BINDING_POLICY=bunch -np $MYPROCS ../../../../lib/sparc -name O2 -log_summary > O2.log
#mpirun -np $MYPROCS ../../../lib/sparc -name Si -log_summary > Si.log
echo "======================================================================="
