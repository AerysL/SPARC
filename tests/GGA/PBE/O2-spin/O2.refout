***************************************************************************
*                           SPARC (May 11, 2020)                          *
*   Copyright (c) 2019 Material Physics & Mechanics Group, Georgia Tech   *
*           Distributed under GNU General Public License 3 (GPL)          *
*                   Start time: Sun May 17 23:25:55 2020                  *
***************************************************************************
                           Input parameters                                
***************************************************************************
CELL: 21 18 18 
LATVEC:
1 0 0 
0 1 0 
0 0 1 
FD_GRID: 105 90 90
FD_ORDER: 12
BC: P P P
KPOINT_GRID: 1 1 1
KPOINT_SHIFT: 0 0 0
SPIN_TYP: 2
ELEC_TEMP_TYPE: Fermi-Dirac
SMEARING: 0.001000000336
CHEB_DEGREE: 35
NSTATES: 10
#WARNING: Number of bands may be insufficient for efficient SCF convergence.
EXCHANGE_CORRELATION: GGA_PBE
CALC_STRESS: 1
CHEFSI_BOUND_FLAG: 0
MAXIT_SCF: 100
MINIT_SCF: 3
MAXIT_POISSON: 3000
TOL_SCF: 1.00E-06
POISSON_SOLVER: AAR
TOL_POISSON: 1.00E-08
TOL_LANCZOS: 1.00E-02
TOL_PSEUDOCHARGE: 1.00E-08
MIXING_VARIABLE: density
MIXING_PRECOND: kerker
TOL_PRECOND: 4.00E-05
PRECOND_KERKER_KTF: 1
PRECOND_KERKER_THRESH: 0.1
MIXING_PARAMETER: 0.3
MIXING_HISTORY: 7
PULAY_FREQUENCY: 1
PULAY_RESTART: 0
REFERENCE_CUTOFF: 0.5
RHO_TRIGGER: 4
FIX_RAND: 0
VERBOSITY: 1
PRINT_FORCES: 1
PRINT_ATOMS: 1
PRINT_EIGEN: 0
PRINT_DENSITY: 0
OUTPUT_FILE: PBE/O2-spin/O2
***************************************************************************
                           Parallelization                                 
***************************************************************************
NP_SPIN_PARAL: 2
NP_KPOINT_PARAL: 1
NP_BAND_PARAL: 10
NP_DOMAIN_PARAL: 1 1 1
NP_DOMAIN_PHI_PARAL: 4 2 3
EIG_SERIAL_MAXNS: 2000
***************************************************************************
                             Initialization                                
***************************************************************************
Number of processors               :  24
Mesh spacing                       :  0.2 (Bohr)
Number of symmetry adapted k-points:  1
Output printed to                  :  PBE/O2-spin/O2.out_1
Total number of atom types         :  1
Total number of atoms              :  2
Total number of electrons          :  12
Atom type 1  (valence electrons)   :  O 6
Pseudopotential                    :  PBE/O2-spin/O.psp8
Pseudocharge radii of atom type 1  :  7.00 7.00 7.00 (x, y, z dir)
Number of atoms of type 1          :  2
Estimated total memory usage       :  1.09 GB
Estimated memory per processor     :  46.50 MB
========================================================================================
                    Self Consistent Field (SCF#1)                     
========================================================================================
Iteration     Free Energy (Ha/atom)    Magnetization     SCF Error        Timing (sec)
1            -1.5852815274E+01         2.0000E+00        1.480E-01        4.066
2            -1.5979827132E+01         2.0000E+00        3.096E-01        1.796
3            -1.5910928498E+01         2.0000E+00        1.091E-01        1.793
4            -1.5910687859E+01         2.0000E+00        8.813E-02        1.746
5            -1.5914340516E+01         2.0000E+00        1.048E-01        1.754
6            -1.5910779661E+01         2.0000E+00        2.887E-02        1.638
7            -1.5910719046E+01         2.0000E+00        2.597E-02        1.697
8            -1.5910494720E+01         2.0000E+00        3.656E-03        1.591
9            -1.5910497780E+01         2.0000E+00        4.706E-03        2.123
10           -1.5910492724E+01         2.0000E+00        1.207E-03        1.494
11           -1.5910492731E+01         2.0000E+00        4.944E-04        1.429
12           -1.5910492867E+01         2.0000E+00        3.001E-04        1.408
13           -1.5910492967E+01         2.0000E+00        1.028E-04        1.410
14           -1.5910492983E+01         2.0000E+00        6.057E-05        1.285
15           -1.5910492978E+01         2.0000E+00        4.834E-05        1.313
16           -1.5910492983E+01         2.0000E+00        1.691E-05        1.367
17           -1.5910492985E+01         2.0000E+00        1.425E-05        1.183
18           -1.5910492998E+01         2.0000E+00        2.100E-06        1.152
19           -1.5910493005E+01         2.0000E+00        1.220E-06        1.118
20           -1.5910492979E+01         2.0000E+00        4.489E-07        0.822
Total number of SCF: 20    

k = [0.000, 0.000, 0.000]
Occupation of state 9 = 0.000000000000000.
Occupation of state 10 = 0.000000000000000.
====================================================================
                    Energy and force calculation                    
====================================================================
Free energy per atom               : -1.5910492970E+01 (Ha/atom)
Total free energy                  : -3.1820985939E+01 (Ha)
Band structure energy              : -6.6119876700E+00 (Ha)
Exchange correlation energy        : -6.6425020955E+00 (Ha)
Self and correction energy         : -6.4063409725E+01 (Ha)
Entropy*kb*T                       : -3.4772595145E-13 (Ha)
Fermi level                        : -2.6406420035E-01 (Ha)
RMS force                          :  1.6391433015E-01 (Ha/Bohr)
Maximum force                      :  1.6391433015E-01 (Ha/Bohr)
Time for force calculation         :  0.028 (sec)
Pressure                           : -7.1434053394E-01 (GPa)
Maximum stress                     :  2.1291011678E+00 (GPa)
Time for stress calculation        :  0.066 (sec)
***************************************************************************
                               Timing info                                 
***************************************************************************
Total walltime                     :  33.418 sec
___________________________________________________________________________

***************************************************************************
*             Material Physics & Mechanics Group, Georgia Tech            *
*                       PI: Phanish Suryanarayana                         *
*                Main Developers: Qimen Xu, Abhiraj Sharma                *
*     Collaborators: J.E. Pask (LLNL), A.J. Medford (GT), E. Chow (GT)    *
*  Acknowledgements: U.S. DOE (DE-SC0019410); U.S. NSF (1333500,1553212)  *
***************************************************************************
                                                                           
