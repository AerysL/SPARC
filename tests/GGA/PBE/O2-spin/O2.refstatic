***************************************************************************
                            Atom positions                                 
***************************************************************************
Fractional coordinates of O:
      0.4285714286       0.5000000000       0.5000000000
      0.5714285714       0.5000000000       0.5000000000
Total free energy (Ha): -3.182098593922724E+01
Atomic forces (Ha/Bohr):
  1.6391433015E-01  -4.2061132878E-08  -4.9992349745E-08
 -1.6391433015E-01   4.2061132878E-08   4.9992349745E-08
Stress (GPa): 
  2.1291011678E+00  -2.2947877511E-06  -3.0781930787E-06 
 -2.2947877511E-06   6.9601714365E-03   3.2020662651E-07 
 -3.0781930787E-06   3.2020662651E-07   6.9602626028E-03
