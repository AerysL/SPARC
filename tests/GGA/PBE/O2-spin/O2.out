***************************************************************************
*                           SPARC (Nov 06, 2019)                          *
*   Copyright (c) 2019 Material Physics & Mechanics Group, Georgia Tech   *
*           Distributed under GNU General Public License 3 (GPL)          *
*                 Start time: Thu Dec 19 14:51:05 2019                    *
***************************************************************************
                           Input parameters                                
***************************************************************************
CELL: 21 18 18 
LATVEC:
1 0 0 
0 1 0 
0 0 1 
FD_GRID: 105 90 90
FD_ORDER: 12
BOUNDARY_CONDITION: 2
KPOINT_GRID: 1 1 1
KPOINT_SHIFT: 0 0 0
SPIN_TYP: 2
ELEC_TEMP_TYPE: Fermi-Dirac
SMEARING: 0.001000000001
CHEB_DEGREE: 35
NSTATES: 10
EXCHANGE_CORRELATION: GGA_PBE
CALC_STRESS: 1
CHEFSI_BOUND_FLAG: 0
MAXIT_SCF: 100
MINIT_SCF: 3
MAXIT_POISSON: 3000
TOL_SCF: 1.00E-06
TOL_POISSON: 1.00E-08
TOL_LANCZOS: 1.00E-02
TOL_PSEUDOCHARGE: 1.00E-08
MIXING_VARIABLE: density
MIXING_PRECOND: none
#WARNING: the preconditioner for SCF has been turned off, this 
#might lead to slow SCF convergence. To specify SCF preconditioner, 
#use 'MIXING_PRECOND' in the .inpt file
MIXING_PARAMETER: 0.3
MIXING_HISTORY: 7
PULAY_FREQUENCY: 1
PULAY_RESTART: 0
REFERENCE_CUTOFF: 0.5
RHO_TRIGGER: 4
FIX_RAND: 0
VERBOSITY: 1
PRINT_FORCES: 1
PRINT_ATOMS: 1
PRINT_EIGEN: 0
PRINT_DENSITY: 0
OUTPUT_FILE: O2
***************************************************************************
                           Parallelization                                 
***************************************************************************
NP_SPIN_PARAL: 2
NP_KPOINT_PARAL: 1
NP_BAND_PARAL: 10
NP_DOMAIN_PARAL: 2 1 1
NP_DOMAIN_PHI_PARAL: 5 2 4
***************************************************************************
                             Initialization                                
***************************************************************************
Number of processors               :  40
Mesh spacing                       :  0.2 (Bohr)
Number of symmetry adapted k-points:  1
Output printed to                  :  O2.out
Total number of atom types         :  1
Total number of atoms              :  2
Total number of electrons          :  12
Atom type 1  (valence electrons)   :  O 6
Pseudopotential                    :  ./O.psp8
Pseudocharge radii of atom type 1  :  7.20 7.20 7.20 (x, y, z dir)
Number of atoms of type 1          :  2
Fractional coordinates of atoms of type 1     :
      0.4285714286       0.5000000000       0.5000000000
      0.5714285714       0.5000000000       0.5000000000
========================================================================================
                    Self Consistent Field (SCF#1)                     
========================================================================================
Iteration     Free Energy (Ha/atom)    Magnetization     SCF Error        Timing (sec)
1            -1.5879236111E+01         2.0000E+00        1.250E-01        9.969
2            -1.5918692391E+01         2.0000E+00        1.739E-01        3.918
3            -1.5918248892E+01         2.0000E+00        1.640E-01        3.862
4            -1.5909620642E+01         2.0000E+00        5.374E-02        3.757
5            -1.5910564582E+01         2.0000E+00        1.634E-02        3.543
6            -1.5910491491E+01         2.0000E+00        6.677E-03        12.645
7            -1.5910492977E+01         2.0000E+00        1.684E-03        3.339
8            -1.5910492660E+01         2.0000E+00        7.750E-04        3.263
9            -1.5910492893E+01         2.0000E+00        3.246E-04        3.114
10           -1.5910492953E+01         2.0000E+00        1.380E-04        2.980
11           -1.5910492972E+01         2.0000E+00        5.046E-05        2.874
12           -1.5910492987E+01         2.0000E+00        2.570E-05        2.790
13           -1.5910492989E+01         2.0000E+00        1.868E-05        2.552
14           -1.5910492979E+01         2.0000E+00        5.716E-06        2.461
15           -1.5910493003E+01         2.0000E+00        4.249E-06        2.463
16           -1.5910493002E+01         2.0000E+00        1.894E-06        2.322
17           -1.5910492996E+01         2.0000E+00        1.027E-06        2.296
18           -1.5910492987E+01         2.0000E+00        4.881E-07        2.099
Total number of SCF: 18    
====================================================================
                    Energy and force calculation                    
====================================================================
Free energy per atom               : -1.5910492985E+01 (Ha/atom)
Total free energy                  : -3.1820985971E+01 (Ha)
Band structure energy              : -6.6119857606E+00 (Ha)
Exchange correlation energy        : -6.6425021347E+00 (Ha)
Self and correction energy         : -6.4063409725E+01 (Ha)
Entropy*kb*T                       : -3.5941697361E-13 (Ha)
Fermi level                        : -2.6402972296E-01 (Ha)
Atomic forces (Ha/Bohr):
  1.6391350822E-01  -1.3266998626E-07  -9.4981834351E-08
 -1.6391350822E-01   1.3266998626E-07   9.4981834351E-08
RMS force                          :  1.6391350822E-01 (Ha/Bohr)
Maximum force                      :  1.6391350822E-01 (Ha/Bohr)
Time for force calculation         :  0.097 (sec)
Stress (GPa) : 
  2.1291012959E+00  -1.5725570440E-06  -1.2098706379E-06 
 -1.5725570440E-06   6.9627004319E-03  -5.2675888349E-07 
 -1.2098706379E-06  -5.2675888349E-07   6.9629993031E-03 
Pressure                           : -7.1434233188E-01 (GPa)
Maximum stress                     :  2.1291012959E+00 (GPa)
Time for stress calculation        :  0.193 (sec)
***************************************************************************
                               Timing info                                 
***************************************************************************
Total walltime                     :  73.052 sec
___________________________________________________________________________

***************************************************************************
*             Material Physics & Mechanics Group, Georgia Tech            *
*                       PI: Phanish Suryanarayana                         *
*                Main Developers: Qimen Xu, Abhiraj Sharma                *
*     Collaborators: J.E. Pask (LLNL), A.J. Medford (GT), E. Chow (GT)    *
*  Acknowledgements: U.S. DOE (DE-SC0019410); U.S. NSF (1333500,1553212)  *
***************************************************************************
                                                                           
